src/06_tuples.py

```diff
@@ -31,14 +31,19 @@ def dist(a, b):
 print("Distance is: {:.2f}".format(dist(a, b)))
 
 
-
 # Write a function `print_tuple` that prints all the values in a tuple
 
 # YOUR CODE HERE
+def print_tuple(z):
+    for i in z: 
```

Comment:
style note: `i` typically indicates an index, in python using for/in assigns the value at the index to your variable.